docs/testrunner-toolkit/saucectl.md

```diff
@@ -1,188 +1,412 @@
 
-> Please refer to the repository [README](https://github.com/saucelabs/saucectl) for installation, development, and contribution requirements.
+See [Installation and Setup](/testrunner-toolkit/installation) for installation options.
 
-## Commands
 
-### `configure`
+## `$ saucectl init [flags]`
 
-__Description__: Sets your [Sauce Labs authentication credentials](https://app.saucelabs.com/user-settings) and generates a `credentials.yml` file. Please refer to [the installation page](/testrunner-toolkit/installation#connecting-to-sauce-labs) for further information.
+A bootstrapping command to generate a configuration file for the framework of your choice in the format required by `saucectl`. Running the command with no parameters initiates interactive mode, prompting you for relevant property values iteratively. Alternatively, you can add supported parameters inline to specify the relevant property values.
+
+In either case, the command generates a `.sauce/config.yml` folder and file in the location from which the command is run. If you have an existing project directory for your framework, it is advised that you run this command from the project root.
+
+```bash title="Interactive Example"
+$ saucectl init
+12:13:20 INF Start Init Command
+? Select region: us-west-1
+? Select framework:  [Use arrows to move, type to filter]
+> cypress
+ playwright
+ puppeteer
+ testcafe
+ espresso
+ xcuitest
+```
+
+```bash title="Batch Example"
+$ saucectl init -r us-west-1 -f cypress -b chrome
+```
+
+`saucectl` supports the following configuration flags as inline specifications.
+
+
+### `--accessKey <string>`
+<p><small>| OPTIONAL | STRING |</small></p>
+
+The authentication access key associated with the Sauce Labs user account making this request. If you have not set your authentication credentials as environment parameters or generated a `credentials.yml` file, this value is required.
+
+**Shorthand:** `-a <string>`
+
+---
+
+### `--app <string>`
+<p><small>| OPTIONAL | STRING | XCUITEST/ESPRESSO ONLY |</small></p>
+
+The path to a valid mobile application to test.
+
+---
+
+### `--artifacts.download.when <string>`
+
+<p><small>| OPTIONAL | STRING |</small></p>
+
+Specifies when and under what circumstances to download artifacts. Valid values are:
+
+* `always`: Always download artifacts.
+* `never`: Never download artifacts.
+* `pass`: Download artifacts for passing suites only.
+* `fail`: Download artifacts for failed suites only. (default value)
+
+---
+
+### `--browserName <string>`
+<p><small>| REQUIRED | STRING | WEB-APP ONLY |</small></p>
+
+The name of the browser in which to run tests.
+
+**Shorthand:** `-b <string>`
+
+---
+
+### `--cypress.config <string>`
+<p><small>| REQUIRED | STRING | CYPRESS ONLY |</small></p>
+
+The file path to the Cypress configuration file (typically `cypress.json`).
+
+---
+
+### `--device ["flag=value"]`
+<p><small>| OPTIONAL | ARRAY | XCUITEST/ESPRESSO ONLY |</small></p>
```

Comment:
```suggestion
### `--device <string>`
<p><small>| OPTIONAL | STRING | XCUITEST/ESPRESSO ONLY |</small></p>
```

IMO, that's better this way.
This is not an array per-se, just a big long string depending the number of flags.

`[]` in cli helps oftenly denotes an optional parameter, whereas `<>` denotes a mandatory one.

Here is a more in-depth explanations: https://ftpdocs.broadcom.com/cadocs/0/CA%20ARCserve%20%20Backup%2015-ENU/Bookshelf_Files/HTML/CMD_Ref/command_line_syntax_characters.htm